MykiBalance.js

```diff
@@ -78,57 +79,71 @@ async function createWidget(card) {
+    let endColor = new Color("#ffffff")
+    let gradient = new LinearGradient()
+    gradient.colors = [startColor, midColor, endColor]
+    gradient.locations = [0.0, 0.72, 0.721]
+    widget.backgroundGradient = gradient
 
     widget.addSpacer()
 
     let mykiUpdateTime = widget.addStack()
-    let updateTxt = mykiUpdateTime.addText("Last updated ")
-    updateTxt.font = Font.boldSystemFont(10)
+    let updateTxt = mykiUpdateTime.addText("")
+    // updateTxt.font = new Font("AppleSDGothicNeo-bold", 10)
+    updateTxt.font = new Font("AppleSDGothicNeo-Light", 10)
     updateTxt.textColor = new Color("#eeeeee")
 
     let updateTime = mykiUpdateTime.addText(dateTxt)
-    updateTime.font = Font.boldSystemFont(10)
+    updateTime.font = new Font("AppleSDGothicNeo-bold", 10)
     updateTime.textColor = new Color("#eeeeee")
 
     let mykiTitle = widget.addStack()
     mykiTitle.centerAlignContent()
 
-    let mykiLogo = mykiTitle.addText("**** " + cardNumber.slice(-4))
-    mykiLogo.font = Font.boldSystemFont(20)
-    mykiLogo.textColor = new Color("#b6e037")
+    let mykiLogo = mykiTitle.addText("•••• " + cardNumber.slice(-5,-1) + " " + cardNumber.slice(-1))
+    mykiLogo.font = new Font("AppleSDGothicNeo-bold", 16)
+    mykiLogo.textColor = new Color("#ffffff")
     mykiTitle.addSpacer()
 
-    let mykiCode = mykiTitle.addText("Top Up >")
-    mykiCode.font = Font.boldSystemFont(20)
-    mykiCode.textColor = new Color("#ffcc00")
+    let mykiCode = mykiTitle.addText("Top Up")
+    let mykiSymbol = mykiTitle.addText(" >")
+    mykiCode.font = new Font("AppleSDGothicNeo-bold", 16)
+    mykiSymbol.font = new Font("AppleSDGothicNeo-bold", 16)
+    mykiSymbol.textColor = new Color("#d92b26")
+    mykiCode.textColor = new Color("#ffffff")
     mykiCode.url = "googlechrome://www.ptv.vic.gov.au/tickets/myki/#topup"
 
     widget.addSpacer()
 
-
     if (mykiPass.length > 0) {
         let middleViewTitle = widget.addStack()
         let moneyTitle = middleViewTitle.addText("myki money")
-        moneyTitle.font = Font.boldSystemFont(12)
+        moneyTitle.font = new Font("AppleSDGothicNeo-bold", 12)
         moneyTitle.textColor = new Color("#eeeeee")
         middleViewTitle.addSpacer()
 
         let passTitle = middleViewTitle.addText("myki pass")
-        passTitle.font = Font.boldSystemFont(12)
+        passTitle.font = new Font("AppleSDGothicNeo-bold", 12)
         passTitle.textColor = new Color("#eeeeee")
     }
 
     let middleView = widget.addStack()
-    let balanceTitle = middleView.addText(balanceTxt)
-    balanceTitle.font = Font.boldSystemFont(40)
-    balanceTitle.textColor = new Color("#eeeeee")
+    let balanceTitleSign = middleView.addText(balanceSign)
+    balanceTitleSign.textColor = new Color("#c2d840")
+    balanceTitleSign.font = new Font("AppleSDGothicNeo-Regular", 30)
+    let balanceTitle = middleView.addText(mykiBalanceValue)
+    balanceTitle.font = new Font("AppleSDGothicNeo-Regular", 30)
```

Comment:
`balanceTitle` uses the font style(size, weight) the same as `daysRemainingTitle` to keep consistent